modules/sql/src/main/scala/EnumRefs.scala

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.sql
+
+/** Minimal classes to refer to other enums by id. This is required to construct
+ *  enums refering to others like Gpi Observing Mode. */
+object EnumRefs {
+  final case class EnumRef[T <: Symbol](tag: String)
```

Comment:
This let's me refer to another enumeration, passing the name of the enumeration as a literal type. I'd rather have the real type but they are not yet available at the time code is generated